lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -445,7 +471,14 @@ def get_students_opened_subsection(request, csv=False):
     students = models.StudentModule.objects.select_related('student').filter(
         module_state_key__exact=module_state_key,
         module_type__exact='sequential',
-    ).values('student__username', 'student__profile__name').order_by('student__profile__name')
+    ).values('course_id','student__id','student__username', 'student__profile__name').order_by('student__profile__name')
+
+    # is there a way to join student_courseaccessrole and courseware_studentmodule
+
+    # here we are using the first record to get course_id.
+    course_key = locator.CourseLocator.from_string(students[0]['course_id'])
```

Comment:
Yep.  This was a hack ;)   I will take a look how django passes the course_id in the request.